.rubocop_todo.yml

```diff
@@ -6,35 +6,20 @@
 # Note that changes in the inspected code, or installation of new
 # versions of RuboCop, may require this file to be generated again.
 
-# Offense count: 3
+# Offense count: 2
```

Comment:
Are the hand edits here the same as we'd get from `rubocop --auto-gen-config --auto-gen-only-exclude` ?

Reply:
nope - it changes the max in some with that option.  that's why I ended up doing them by hand.